verifier/src/identity.rs

```diff
@@ -299,96 +328,212 @@ where
 /// identity, while the `TrustedIdentitiesVerifier` is for verifying one of *multiple* identities.
 #[derive(Debug, Clone)]
 enum TrustedIdentityVerifier {
-    MrEnclave(TrustedMrEnclaveIdentityVerifier),
-    MrSigner(TrustedMrSignerIdentityVerifier),
+    MrEnclave(And<MrEnclaveVerifier, AdvisoriesVerifier>),
+    MrSigner(And<MrSignerVerifier, AdvisoriesVerifier>),
+}
+
+impl TrustedIdentityVerifier {
+    fn fmt_padded(
+        &self,
+        f: &mut Formatter<'_>,
+        pad: usize,
+        identity: &IdentityOutput,
+    ) -> core::fmt::Result {
+        // We re-verify here, in order to leverage the `fmt_padded` method on the `Verifier` trait
+        match self {
+            Self::MrEnclave(verifier) => {
+                let result = verifier.verify(identity);
+                verifier.fmt_padded(f, pad, &result)
+            }
```

Comment:
I couldn't think of another way to get the output without re-verifying. I don't think we'll have that many identities and the overall comparisons should be pretty cheap.  And this will only be done when displaying the results.